### PR TITLE
[#169618545] Add new required settings to functions-app app

### PR DESCRIPTION
**Pull request by francescopersico**

### CODE REVIEW

File: dev/westeurope/function_app_app/terraform.tfvars [hunk 1]
```diff
@@ -61,22 +61,51 @@ functionapp_settings = [
...
+    value = "Presidenza del Consiglio dei Ministri"
+  },
+  {
+    name  = "MAIL_SENDER_SERVICE"
+    value = "Team per la Trasformazione Digitale<br />Progetto IO"
+  },
+  {
+    name  = "FUNCTIONS_PUBLIC_URL"
+    # TODO: Change this when io-functions-public will be deployed
+    value = "https://localhost/public"
```
Comment (LucaPrete) [inline dev/westeurope/function_app_app/terraform.tfvars]: What's the reason for not setting this properly since the beginning?

Comment (francescopersico) [reply dev/westeurope/function_app_app/terraform.tfvars]: The function app functions-public does not exist yet.

Comment (LucaPrete) [reply dev/westeurope/function_app_app/terraform.tfvars]: Can we add that first?

Comment (francescopersico) [reply dev/westeurope/function_app_app/terraform.tfvars]: The code of the new function-app is not completed so we can't.

Comment (LucaPrete) [reply dev/westeurope/function_app_app/terraform.tfvars]: Are the new secret already in place instead?

Comment (francescopersico) [reply dev/westeurope/function_app_app/terraform.tfvars]: Yes i have added fn2CommonsMailupUsername and fn2CommonsMailupSecret to io-dev-keyvault.

**Pull request merged**